src/modules/reading-room/components/AddToCollectionBlade/AddToCollectionBlade.tsx

```diff
@@ -229,6 +235,7 @@ const AddToCollectionBlade: FC<AddToCollectionBladeProps> = (props) => {
 
 					<span className={styles['c-add-to-collection-blade__list-item__label']}>
 						{collection?.name}
+						{collection?.objects?.length}
```

Comment:
count is concatenated to the folder name:
![image](https://user-images.githubusercontent.com/1710840/164736147-ac16f6c8-c285-4c14-9f24-71685ae09ff7.png)